packages/core/template_project/app/build.gradle

```diff
@@ -196,5 +196,11 @@ dependencies {
     <% for(const dep of buildGradle.dependencies) { %>
         implementation '<%= dep %>'
     <% } %>
-    implementation 'com.google.androidbrowserhelper:androidbrowserhelper:2.0.0'    
+     if (twaManifest.alphaFeatures && twaManifest.alphaFeatures.enabled) {
+      this.buildGradle.dependencies.add(
+          'com.google.androidbrowserhelper:androidbrowserhelper:1.4.0-alpha01');
```

Comment:
I'm not 100% sure how this works, maybe @andreban can help me, but I don't think this will add the lines to build.gradle.

We're adding the dependencies to the list of dependencies, but we added the list of dependencies to the file up on line 196-198. So we're adding this *after* we've read from the list.

Reply:
I'm surprised running `init` or `update` on this template works and would expect an error on `twaManifest.alphaFeatures`, as `twaManifest` would be undefined. The code should look like something like this:
```
<% if (alphaFeatures && alphaFeatures.enabled { %>
    implementation 'com.google.androidbrowserhelper:androidbrowserhelper:1.4.0-alpha01' 
<% } else  { %> 
    implementation 'com.google.androidbrowserhelper:androidbrowserhelper:2.0.0' 
<% } %> 
```